modern/base/Dockerfile

```diff
@@ -97,12 +97,10 @@ RUN apt-get -qq update \
     && runuser -u conan -- printf 'eval "$(pyenv init -)"\n' >> /home/conan/.bashrc \
     && runuser -u conan -- printf 'eval "$(pyenv virtualenv-init -)"\n' >> ~/.bashrc \
     && chown -R conan:1001 /opt/pyenv \
-    && chown -R conan:1001 /opt/conan
+    && chmod +x /opt/conan/conan_bootstrap.sh \
+    && chown -R conan:1001 /opt/conan 
 
 USER conan
 WORKDIR /home/conan
 
-RUN pip install conan==${CONAN_VERSION} \
-    && CONAN_REVISIONS_ENABLED=1 CONAN_USER_HOME=/tmp/conan conan install -r conancenter -g deploy -if /opt/conan -l /opt/conan/conan.lock /opt/conan/conanfile.txt \
-    && rm -rf /tmp/conan \
-    && conan config set general.revisions_enabled=1
+RUN /opt/conan/conan_bootstrap.sh ${CONAN_VERSION}
```

Comment:
We can keep it all here:

```
pip install conan~=1.0
CONAN_REVISIONS_ENABLED=1 CONAN_USER_HOME=/tmp/conan conan install -r conancenter -g deploy -if /opt/conan -l /opt/conan/conan.lock /opt/conan/conanfile.txt
rm -rf /tmp/conan

pip install conan==${CONAN_VERSION}
```

And move the `conan config set general.revisions_enabled=1` to the same place where we initiate the profile and so on, to the _deploy_ images.